src/runtime/virtcontainers/mount.go

```diff
@@ -208,7 +208,7 @@ func getDeviceForPath(path string) (device, error) {
 
 var blockFormatTemplate = "/sys/dev/block/%d:%d/dm"
 
-var checkStorageDriver = isDeviceMapper
+var checkStorageDriver = checkBlockDevice
```

Comment:
`isDeviceMapper` is left over but it's a piece of dead code now? And the new code is not unit-tested as the code pieces.

Reply:
+1 for removing the isDeviceMapper code if not used